tests/test-extensions/heisenberg-extension/src/main/java/org/mule/test/heisenberg/extension/HeisenbergExtension.java

```diff
@@ -100,7 +100,7 @@
   private int start = 0;
   private int stop = 0;
   private int dispose = 0;
-  public static int sourceTimesStarted = 0;
+  public static volatile Integer sourceTimesStarted = 0;
```

Comment:
this can be an `int` with the `volatile`, or an `AtomicInteger` directly if not.